test/test-executor.js

```diff
@@ -54,6 +59,8 @@ tap.test('executor', function(t) {
       Container: Container,
       basePort: 4000,
       control: 'http://token@host:66',
+      driver: 'direct',
+      svcAddr: 'host',
```

Comment:
Right, I meant to pick a better name.. unfortunately ssh over LTE is painfully non-interactive.